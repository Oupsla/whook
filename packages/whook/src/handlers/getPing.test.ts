import initGetPing from './getPing';

describe('getPing', () => {
  it('should work', async () => {
    const getPing = await initGetPing({
      NODE_ENV: 'test',
    });
    const response = await getPing();

    expect({
      response,
    }).toMatchSnapshot();
  });
});
