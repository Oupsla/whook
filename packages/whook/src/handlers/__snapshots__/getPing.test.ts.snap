// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getPing should work 1`] = `
Object {
  "response": Object {
    "body": Object {
      "pong": "pong",
    },
    "headers": Object {
      "X-Node-ENV": "test",
    },
    "status": 200,
  },
}
`;
